src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -142,15 +154,19 @@ - (void)setExpirationForMessage:(TSMessage *)message
     [self setExpirationForMessage:message expirationStartedAt:[NSDate ows_millisecondTimeStamp]];
 }
 
-+ (void)setExpirationForMessage:(TSMessage *)message expirationStartedAt:(uint64_t)expirationStartedAt
+- (void)setExpirationForMessage:(TSMessage *)message expirationStartedAt:(uint64_t)expirationStartedAt
 {
-    dispatch_async(self.serialQueue, ^{
```

Comment:
Why is this no longer async? And why do we no longer want it on a serialized queue?